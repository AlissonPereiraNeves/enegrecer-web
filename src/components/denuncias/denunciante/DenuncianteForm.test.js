import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Denunciante from './DenuncianteForm';

describe('DenuncianteForm', () => {
  it('deve seguir a estrutura definida ', () => {
    const handleChange = jest.fn();
    const denunciante = shallow(<Denunciante handleChange={handleChange} />);
    const denuncianteJson = toJson(denunciante);
    expect(denuncianteJson).toMatchSnapshot();
  });
});
