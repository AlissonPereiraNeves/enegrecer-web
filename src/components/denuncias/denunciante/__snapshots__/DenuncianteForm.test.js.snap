// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DenuncianteForm deve seguir a estrutura definida  1`] = `
<div>
  <br />
  <h4>
    Você poderia nos passar algumas informações suas?
  </h4>
  <br />
  <p>
     Não se preocupe sua identidade será mantida em sigilo. Essas informações são importantes para futuramente darmos um retorno das medidas que estão sendo tomadas contra esse agressor. 
  </p>
  <div
    className="row"
  >
    <nome
      id="nome-vitima"
      onChange={[Function]}
      state="nomeDenunciante"
    />
    <campoTexto
      divClasse="input-field col s6"
      id="email-denunciante"
      label="Email"
      maxLen={40}
      state="emailDenunciante"
      type="text"
    />
  </div>
  <div
    className="row"
  >
    <telefone
      divClasse="col s6"
      id="telefone-denunciante"
      label="Telefone"
      state="telefoneDenunciante"
    />
    <combobox
      divClasse="col s6"
      id="cidade-denunciante"
      itens={
        Array [
          "Belo Horizonte",
        ]
      }
      label="Cidade *"
      state="cidadeDenunciante"
      valorPadrao=""
    />
  </div>
  <div
    className="row"
  >
    <genero
      divClasse="col s6"
      id="genero-denunciante"
      state="generoDenunciante"
    />
    <Data
      divClasse="input-field col s6"
      id="data-nascimento-denunciante"
      label="Data de Nascimento"
      state="dataNascimentoDenunciante"
    />
  </div>
  <div
    className="row"
  >
    <comboRaca
      classes="col s6"
      divClasse=""
      id="raca-denunciante"
      somenteRacasVitima={true}
      state="racaDenunciante"
    />
  </div>
</div>
`;
